.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+aliases:
+  - &restore-cache
+    keys:
+      - dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+      # Fallback in case checksum fails
+      - dependencies-{{ .Branch }}-
+
+  - &save-cache
+    paths:
+      - node_modules
+      - website/node_modules
+    key: dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+
+  - &yarn-install
+    run: |
+      sudo npm i -g yarn@^0.27.5
```

Comment:
If we do want to upgrade Yarn, we should do that in a Docker image that's layered on top of another one  😃 

We should at least be installing Yarn using the Debian package if the image is Debian, based, too :D 

Feel free to reach out to me if you need help with it!